pages.pt-BR/common/ab.md

```diff
@@ -0,0 +1,20 @@
+# ab
+
+> Ferramenta de _benchmarking_ do servidor Apache. A ferramenta mais simples para realizar testes de carga.
```

Comment:
```suggestion
> Ferramenta de benchmarking do servidor Apache. A ferramenta mais simples para realizar testes de carga.
```
I'd suggest removing those underscores surrounding "benchmark", there's no need to emphasize the word.

Reply:
:+1: